mlx-rs/src/ops/convolution.rs

```diff
@@ -191,6 +191,149 @@ pub fn conv3d_device(
     }
 }
 
+/// 1D transposed convolution over an input with several channels.
+///
+/// > Only the default `groups=1` is currently supported.
+///
+/// # Params
+///
+/// - array: input array of shape `[N, H, C_in]`
+/// - weight: weight array of shape `[C_out, H, C_in]`
+/// - stride: kernel stride
+/// - padding: input padding
+/// - dilation: kernel dilation
+/// - groups: input feature groups
+/// - stream: stream or device to evaluate on
+#[default_device]
+pub fn conv_transposed1d_device(
+    array: &Array,
+    weight: &Array,
+    stride: impl Into<Option<i32>>,
+    padding: impl Into<Option<i32>>,
+    dilation: impl Into<Option<i32>>,
+    groups: impl Into<Option<i32>>,
+    stream: impl AsRef<Stream>,
+) -> Result<Array, Exception> {
+    let stride = stride.into().unwrap_or(1);
```

Comment:
We should probably mention the default values in the doc (should probably do the same for `conv_transpose2d` and `conv_transpose3d`